utils/build-dpkg.sh

```diff
@@ -223,7 +223,7 @@ cat << EOF >> $CONTROL_FILE
 
 Package: librpmem
 Architecture: any
-Depends: libfabric (>= $LIBFABRIC_MIN_VERSION), \${shlibs:Depends}, \${misc:Depends}
+Depends: libfabric-dev (>= $LIBFABRIC_MIN_VERSION), \${shlibs:Depends}, \${misc:Depends}
```

Comment:
Binary packages shouldn't depend on development packages. The binary package of libfabric in Debian is called libfabric1 but specifying this is not necessary because shlibs:Depends already contains it. So the correct thing to do is to remove the explicit dependency on the libfabric binary package.

However, I realized that build-time dependency on libfabric-dev is missing, so I think the spec should also have Build-Depends to libfabric-dev (with >= LIBFABRIC_MIN_VERSION).